packages/react-components/react-menu/library/src/components/MenuItem/useMenuItem.tsx

```diff
@@ -104,7 +115,38 @@ export const useMenuItem_unstable = (props: MenuItemProps, ref: React.Ref<ARIABu
       elementType: 'span',
     }),
     secondaryContent: slot.optional(props.secondaryContent, { elementType: 'span' }),
+    subText: slot.optional(props.subText, { elementType: 'span' }),
   };
   useCharacterSearch(state, innerRef);
   return state;
 };
+
+/**
+ * MenuSplitGroup needs to apply extra styles when its main item is in multiline layout mode
+ * Notify the parent MenuSplitGroup so that it can handle this case
+ */
+const useNotifySplitItemMultiline = (options: { hasSubmenu: boolean; subText: MenuItemProps['subText'] }) => {
```

Comment:
nit: 

```suggestion
const useNotifySplitItemMultiline = (options: { hasSubmenu: boolean; isMultiline: boolean }) => {
```

as you don't need the slot itself